src/content/docs/patch-signing.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: Patch Signing (preview)
+description: How to sign patches
+---
+
+{/* cspell:words outform pubout */}
+
+:::note
+We've shipped this preview of patch signing to unblock certain customer
+evaluations. While we're confident that this preview adds a useful layer of
+security, it does not yet have all the expected integrations with cloud signing,
+keychains, etc. that a final product would expect to have. We also do not yet
```

Comment:
```suggestion
keychains, etc, that a final product would expect to have. We also do not yet
```

Reply:
"etc." should be followed by a period as it is an abbreviation.